README.md

```diff
@@ -23,13 +23,22 @@ SnakeYAML Engine KMP is based on
 The library has been successfully ported to KMP, and is consumed by [kaml](https://github.com/charleskorn/kaml),
 a library that integrates with [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization).
 
-SnakeYAML Engine KMP uses comprehensive [YAML test suites](https://github.com/yaml/yaml-test-suite) that validate correctness. However, performance has not been evaluated.
+SnakeYAML Engine KMP uses comprehensive [YAML test suites](https://github.com/yaml/yaml-test-suite)
+that validate correctness. However, performance has not been evaluated.
 
 ## How to use
 
-The library is published to Maven Central. Use the following Maven coords:
+> [!NOTE]
+> SnakeYAML Engine KMP is a low-level library, only intended to be used by libraries.
```

Comment:
It also has some higher-level API. I'd omit this change here. We already mention kaml earlier.

Reply:
Yeah, I see your point.

My intention was to add a note to guide the majority of devs towards a more straightforward solution, because, while it's true that SnakeKMP _can_ be used at a high level, it does requires more set up.

What about this instead? Something that's more of a comparative, just providing some information for people to use.

> [!TIP]
> Are you wondering if SnakeYAML Engine KMP is for you?
>
> SnakeYAML Engine KMP offers fine-grained control and advanced functionality, which is a good fit if you need more customization and flexibility.
> 
> If you're for a hassle-free YAML parsing experience, we recommend using kaml, which is designed for simplicity and ease of integration.

Reply:
Much better!